README.md

```diff
@@ -1,7 +1,7 @@
-# Membrane Multimedia Framework: File Element
+# Membrane File plugin
 
-[![Hex.pm](https://img.shields.io/hexpm/v/membrane_element_file.svg)](https://hex.pm/packages/membrane_element_file)
-[![CircleCI](https://circleci.com/gh/membraneframework/membrane-element-file.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane-element-file)
+[![Hex.pm](https://img.shields.io/hexpm/v/membrane_file_plugin.svg)](https://hex.pm/packages/membrane_file_plugin)
+[![CircleCI](https://circleci.com/gh/membraneframework/membrane_file_plugin.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_file_plugin)
 
 This package provides elements that can be used to read from and write to files.
```

Comment:
In most repos we have removed sentence `This package provides elements that...`. Maybe replace it with sth like `Plugin for reading from and writing to files.`?